CRFModuleValidation/CRFMotionRecorder.swift

```diff
@@ -37,13 +37,41 @@ import ResearchSuite
 
 public enum CRFMotionRecorderType : String, Codable {
     
+    /// Raw accelerometer reading.
     case accelerometer
+    
+    /// Raw gyroscope reading.
+    case gyro
+
+    /// Raw magnetometer reading.
+    case magnetometer
+    
+    /// Calculated orientation of the phone using the gyro and magnetometer (if appropriate)
+    ///
+    /// - note: If the `magneticField` is included in the configuration's list of desired
+    /// recorder types then the reference frame is `.xMagneticNorthZVertical`. Otherwise,
+    /// the motion recorder will use `.xArbitraryZVertical`.
     case attitude
+    
+    /// Calculated vector for the direction of gravity in the coordinates of the phone.
+    /// This is included in the `CMDeviceMotion` data object.
     case gravity
-    case gyro
+    
+    /// The magnetic field of the device for devices with a gyro that is included in the
```

Comment:
should it say "devices with a magnetometer"?

Reply:
oops.